src/Main.cc

```diff
@@ -225,12 +231,6 @@ int main(int argc, char* argv[])
   cmd_argc = argc;
   cmd_argv = argv;
 #ifdef ENABLE_PYTHON3
-  // Capture the current SIGINT handler before python changes it.
-  struct sigaction orig_sigint_handler;
-  sigaction(SIGINT, NULL, &orig_sigint_handler);
```

Comment:
Why is this no longer needed?

Reply:
tried to clarify this in the commit message (but I now realize that wasn't really explicit, sorry!).

The python initialization logic (which is the code path altering the signal) is now only triggered when the `-python` flag is passed, and since the python code path return from `main` there shouldn't be anything to restore in either cases.

Reply:
I'm not sure what you mean by "the python code path return from main".  What happens in the event of a SEGV?

Reply:
> I'm not sure what you mean by "the python code path return from main

I agree this was not very precise, what I meant is that the code path that handle the case where the `-python` flag is present returns from the `main` function, effectively existing the `openroad` program.

> What happens in the event of a SEGV?

The default Python Interpreter code path doesnt' seems to trap `SIGSEGV`:
- https://github.com/python/cpython/blob/342b1151ae7e6ae849c1ed7c8a2cbfdb4edcf51c/Modules/signalmodule.c#L1948 
- https://github.com/python/cpython/blob/d291a82df33cd8c917a374fef2a2373beda78b77/Modules/main.c#L653

Reply:
Please test that assumption.  In the past I added that code because python does install its own handler which we don't want to override ours.

Reply:
Yes, but the looking at the previous code it seemed that you only restored those handler if the `-python` was not passed (and I assumed this was because `initPython` was inconditionally called).

Was there a good reason we called `initPython` even if `-python` was not passed?

Reply:
python_cmd allows you to call python from tcl.  It was never documented but was more for testing.  We could drop it.

Reply:
ah that all make sense now! happy to restore the old code path if needed.

Reply:
My only care is that you get a proper stack trace on error whether in tcl or python.  Feel free to remove python_cmd if that makes things simpler.